brave/brave-core
# Add chrome://rewards-internals to assist with support

Fixes brave/brave-browser#1174

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:

- Start Brave
- Navigate to chrome://version
- Click the "More details about Brave Rewards" link to go to chrome://rewards-internals
- Verify that information looks correct

## Reviewer Checklist:
